app/models/user.rb

```diff
@@ -12,6 +12,13 @@ class User < ActiveRecord::Base
 
   has_secure_password
   enum role: %w(default admin)
+  
+  has_attached_file :avatar, styles: {thumb: '100x100>',
```

Comment:
Space inside { missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.